docs/content-blocks.md

```diff
@@ -84,6 +84,17 @@ but I'll stop here.
 ```
 ````
 `````
+## Examples of marginnote and sidenotes
```

Comment:
could you move this underneath "sidebars" since that's the section that covers things like margin notes?

Reply:
I'll juts give this a shot